Add Learn Architecture section and related pages. (#624)
AstarNetwork/astar-docs: docs/learn/architecture/astar-zkevm.md

---
title: Astar zkEVM
sidebar_position: 2
---

# Overview

> The Astar zkEVM is a Layer 2 scaling solution for Ethereum connected to Polygon Aggregation Layer (AggLayer) for shared interoperability and data availability, otherwise known as a Validium. Smart contracts that run on the Ethereum Virtual Machine also run on Astar zkEVM. The zkEVM boasts the following key features:

Reviewer: ```suggestion
The Astar zkEVM is a Layer 2 scaling solution for Ethereum connected to Polygon Aggregation Layer (AggLayer) for shared interoperability. It's a modular Validium blockchain. Smart contracts that run on the Ethereum Virtual Machine also run on Astar zkEVM. The zkEVM boasts the following key features:
```

> - **Higher TPS than Ethereum or Astar Substrate EVM** - Leveraging zk rollup architecture, transactions are parallelized on Layer 2 and submitted on-chain to Layer 1 in batches, supercharging web3 games and DeFi applications requiring high performance.

Reviewer: This is wrong. Astar zkEVM doesn't have a higher TPS.

- **Lower Transaction Fees compared to Ethereum** - Due to the transaction batching, as explained above.
- **Full EVM-equivalence** - Not only EVM compatibility; Equivalence. Smart contracts that work on Ethereum also work on Astar zkEVM. See the [smart contract section](/docs/build/zkEVM/smart-contracts/) for more information.
- **Native Account Abstraction** - The Astar zkEVM provides native features designed to revolutionize the end-user experience, and make it seamless. See the [Account Abstraction section](/docs/build/zkEVM/integrations/account-abstraction/) to learn more about how to refine the end-user experience.

> - **Recognized Partners** - Established names and brands that developers trust power the Astar zkEVM. See the [integrations section](/docs/build/zkEVM/integrations/) for more information about 3rd party service providers.

Reviewer: The term *partners* shouldn't be used. Partners means 50% and 50% liability.
